eng/scripts/create_coverage.ps1

```diff
@@ -0,0 +1,17 @@
+#Requires -Version 7.0
+
+$coverageFiles = [Collections.Generic.List[String]]@()
+Get-Childitem -recurse -path $(SCOPE) -filter coverage.txt | foreach-object {
```

Comment:
nit: I don't care much for the powershell casing style, but I'd pick one to be consistent (e.g. capitalize ForEach-Object). Also it's `Get-ChildItem`.

Reply:
I'll change to the consistent uppercase style, didn't notice this when I copied it over from the yml